debug-service-in-chrome.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+
+# Check if a port number is provided
+if [ -z "$1" ]; then
+  echo -e "\033[31mUsage: $0 <port>\033[0m"
+  exit 1
+fi
+
+PORT=$1
+
+# Get the PID of the process listening on the provided port
+PID=$(lsof -i:$PORT -t)
+
+# Check if the PID was found
+if [ -z "$PID" ]; then
+  echo -e "\033[31mNo process found listening on port $PORT.\033[0m"
+  exit 1
+else
+  # Send the SIGUSR1 signal to the process
+  kill -SIGUSR1 $PID
+  echo -e "\033[32mSent SIGUSR1 to process with PID $PID.\033[0m"
+fi
+
+# Open Chrome with the specified URL
+open -a "Google Chrome" "chrome://inspect/#devices"
+
+# Display information message
+echo -e "\033[34m====================================================\033[0m"
+echo -e "\033[33mThis script helps you debug a Node.js application running on http://localhost:$PORT in Google Chrome DevTools.\033[0m"
+echo -e "\033[34m====================================================\033[0m"
+echo -e "\033[36m1. Once the DevTools are open, click 'inspect' below the 'Remote Target' link.\033[0m"
```

Comment:
This message could be a bit better? Like to find the remote target

Reply:
can you help me by completing a full info message here?

Reply:
I was actually trying to open the devtools directly from the client's devtools. Thus, it didn't match the instruction. This message seems okay.